src/components/ContactsSection.tsx

```diff
@@ -0,0 +1,48 @@
+import { SectionData } from "./types";
+
+type ContactsSectionProps = {
+  contactsSectionData: SectionData;
+};
+
+export const ContactsSection = ({
+  contactsSectionData,
+}: ContactsSectionProps) => {
+  if (!contactsSectionData || !contactsSectionData.content) return null;
+
+  const breakBeforeWords = [
+    "Адреса",
+    "Телефон",
+    "Електронна",
+    "Графік",
+    "Завітайте",
+  ];
+  const listBeforeWords = ["Адреса", "Телефон", "Електронна", "Графік"];
+
+  let formattedContent = contactsSectionData.content.replace(
+    new RegExp(`(${breakBeforeWords.join("|")})`, "g"),
+    "\n$1"
+  );
+
+  formattedContent = formattedContent.replace(
+    new RegExp(`(${listBeforeWords.join("|")})`, "g"),
+    "● $1"
+  );
```

Comment:
The code may look better:

```suggestion
 const formattedContent = content
    .replace(new RegExp(`(${BREAK_BEFORE_WORDS.join("|")})`, "g"), "\n$1")
    .replace(new RegExp(`(${LIST_BEFORE_WORDS.join("|")})`, "g"), "● $1");
```